.github/workflows/cypress-firefox.yml

```diff
@@ -0,0 +1,22 @@
+name: Cypress push and pull
+
+on: [push, pull_request]
+
+jobs:
+  cypress-run:
+     name: firefox
+     runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
```

Comment:
One reason the action is failing is because the dependencies haven't been installed yet.

Eventually we'll probably want to clean this up and move to `npm`, but since the repo is already using `yarn`, here's how you can install everything:

```
      - name: Install Yarn
        run: |
          npm install -g yarn

      - name: Get yarn Cache Directory Path
        id: yarn-cache-dir-path
        run: echo "::set-output name=dir::$(yarn cache dir)"

      - uses: actions/cache@v2
        # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
        with:
          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
          restore-keys: |
            ${{ runner.os }}-yarn-

      - name: Install Dependencies
        run: |
          yarn install && apt update && apt install -y firefox libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
```

To get things running locally with `act`, I had to install the Firefox browser and some missing Cypress dependencies, notably xvfb.

I believe the GitHub Action containers already have both Chrome and Firefox installed, and along with xvfb and the other dependencies. So for your PR, you can probably just install the dependencies in `package.json` with this:

```
      - name: Install Dependencies
        run: yarn install
```